data/test_endpoints/example_endpoints.yml

```
@@ -14,6 +14,7 @@ tracker_store:
   port: 6379
   db: 0
   password: password
+  prefix: xyz
```

this should be a useful example, how about
```suggestion
  prefix: conversation
```